Even_Odd.java

```diff
@@ -0,0 +1,15 @@
+
+
+import java.util.Scanner;
+public class Even_Odd {
```

Comment:
Do not write `_` in class name try this https://en.wikipedia.org/wiki/Camel_case